docs: add kapa section to readme (#7347)
spectrocloud/librarium: README.md

Your page will now redirect to `/integrations/packs/?pack=myPack`.

## Our Kapa Widget Setup

Kapa is our embedded AI assistant that helps users navigate documentation and find answers through natural-language
search. It provides a plug-and-play JavaScript script that loads the Kapa widget with a solid list of built-in

> customization options that come out-of-the-box. It also has a JavaScript API that we can use to build customized

Reviewer: ```suggestion
customization options that come out of the box. It also has a JavaScript API that we can use to build customized
```
This is according to Grammarly

functions if we need it, but a custom-built solution has high upfront as well as maintenance costs.

### Configuration Block

Since we use Docusaurus, which Kapa has built dedicated customization for, all we did to enable Kapa was to add a
configuration block in our `docusaurus.config.js` file. You can find the relevant block in the `allScripts` variable.

### Customizations

All of our customizations, except for the keyboard shortcut, were made using Kapa’s supported configuration fields.
While additional changes using CSS or JavaScript are technically possible, Kapa strongly discourages unsupported
customizations due to maintenance and compatibility risks with future updates. You can review all available parameters
in the [Kapa documentation - Configuration](https://docs.kapa.ai/integrations/website-widget/configuration) page.

#### Branding

> We use fields such as `"data-project-color"` and `"data-project-logo"` to customize the color and logo of the website

Reviewer: ```suggestion
We use fields such as `data-project-color` and `data-project-logo` to customize the color and logo of the website
```
I'm not sure about this one, but I would write it like this, the parameters are called _data-project-color_ and _data-project-logo_, quotation marks are user for syntax.

> widget and the logo it displays.

Reviewer: Are these two different logos: "and logo of the website widget and the logo it displays"?

```json
...
    "data-project-color": "#3A9D99",
    "data-project-logo": "/img/spectrocloud-mark-light-bkgd-RGB.svg"
```

#### Pop-up Style

> Instead of having a central pop-up that blocks the rest of the site, we chose a dialogue window that appears on the side
> which allows users to navigate to different pages while continuing the conversation. We did this by following the

Reviewer: ```suggestion
Instead of having a central pop-up that blocks the rest of the site, we chose a dialogue window that appears on the side.
It allows users to navigate to different pages while continuing the conversation. We did this by following the
```

Reviewer: ```suggestion
Instead of having a central pop-up that blocks the rest of the site, we chose a dialogue window that appears on the side
allowing users to navigate to different pages while continuing the conversation. We did this by following the
```

I think this sentence is a little bit long, I provided several options how to make it shorter, if you'd like to.

instructions on
[Style the widget like a customer help chat](https://docs.kapa.ai/integrations/website-widget/tutorials/style-widget-like-help-chat).
The entire implementation also only required changes to the configuration block.

#### Privacy Notice

These fields are part of Kapa's built-in consent flow and can be set directly in the config block. We used the following

> blocks to implement the privacy notice to meet compliance requirements. The text was provided by our compliance team.

Reviewer: ```suggestion
blocks to implement the privacy notice in accordance with compliance requirements. Our compliance team provided the text.
```

If you'd like to avoid saying "to implement... to meet" and passive voice

```json
...
    "data-consent-required": "true",
    "data-consent-screen-title": "Privacy Notice",
    "data-consent-screen-disclaimer":
      "Thank you for using our chat service!  Information you submit through this chat is subject to our [Privacy Policy and Terms of Use](https://www.spectrocloud.com/privacy-policy) and will be processed by our service provider. Please do not enter sensitive information. Chat transcripts may be kept for future reference.",
    "data-consent-screen-accept-button-text": "Accept",
    "data-consent-screen-reject-button-text": "Decline",
...
```

#### Keyboard Shortcut

> We use the `/static/scripts/kapa-shortcut.js` file to implement a keyboard shortcut for users to open the Kapa widget
> with the `Ctrl + I` keyboard shortcut. We wanted to use `Ctrl + K`, but that space was taken up by Algolia.

Reviewer: ```suggestion
We use the `/static/scripts/kapa-shortcut.js` file to enable the **CTRL + I** keyboard shortcut for opening the Kapa widget. We wanted to use **CTRL + K**, but that space was taken up by Algolia.
```
I suggest this to avoid repeating "keyboard shortcut".
And we are also supposed to use bold and all caps for keyboard combinations according to the [style guide](https://spectrocloud.atlassian.net/wiki/spaces/DE/pages/1765933057/Spectro+Cloud+Internal+Style+Guide).
